content/docs/dvclive/resume.md

```diff
@@ -0,0 +1,40 @@
+# Resume training
+
```

Comment:
Similar to what @jorgeorpinel said  in https://github.com/iterative/dvc.org/pull/3923#discussion_r965222711, I still think we need more motivation and a high-level summary here, like "DVCLive can enable DVC checkpoints at each step and resume from any previous step."

It would be great if this page could walk through a full example of an experiment with checkpoints and resume.

Reply:
I would also say that Resume and Outputs look like a lower level sections compared to everything else. 

In DVC they would belong to the User Guide, here I'm not sure.

Especially this one "Resume" feel out-of-the-blue and not clear why we have it, etc

Reply:
Will try to convert it into a more complete guide for resuming training. 
It is just info that is needed in order to properly resume training and didn't fit anywhere else

Reply:
> It would be great if this page could walk through a full example of an experiment with checkpoints and resume.

I ended up dropping the page 👼 . I wrote the guide but it was too much of a duplicate of the existing checkpoints guide https://dvc.org/doc/user-guide/experiment-management/checkpoints#registering-checkpoints-in-your-code. 

For non-checkpoints, I believe we should address proper resuming in DVCLive per https://github.com/iterative/dvclive/issues/140 and document inside each individual framework page.